src/components/NewMeasure/CreateNewMeasureDialog.tsx

```diff
@@ -138,6 +139,23 @@ const CreateNewMeasureDialog = ({ open, onClose }) => {
     setFocusedField(field);
   };
 
+  useEffect(() => {
+    if (
+      (formik.touched["cqlLibraryName"] || focusedField === "cqlLibraryName") &&
+      formikErrorHandler("cqlLibraryName", true)
+    ) {
+      setCqlLibraryHelper(
+        "Measure Library name must start with an upper case letter, followed by alpha-numeric character(s) and must not contain spaces or other special characters."
+      );
+    } else if (formik.values.cqlLibraryName.length > 64) {
+      setCqlLibraryHelper(
+        "A Measure Libary Name cannot be more than 64 characters"
+      );
+    } else {
+      setCqlLibraryHelper(null);
+    }
+  }, [formik.values.cqlLibraryName, formik.touched, focusedField]);
```

Comment:
could this be moved to validator schema?

Reply:
done!

Reply:
I don't think that code got added to the PR